ironfish/src/wallet/account/encoder/mnemonic.ts

```diff
@@ -14,7 +14,7 @@ import { ACCOUNT_SCHEMA_VERSION } from '../account'
 import { AccountDecodingOptions, AccountEncoder, AccountEncodingOptions } from './encoder'
 
 export class MnemonicEncoder implements AccountEncoder {
```

Comment:
`options` is required for the `MnemonicEncoder` since `language` is specifically related to the mnemonic.